src/pandablocks_ioc/ioc.py

```diff
@@ -893,11 +893,13 @@ def _make_pos_out(
         # Create the POSITIONS "table" of records. Most are aliases of the records
         # created above.
         positions_record_name = f"POSITIONS:{self._pos_out_row_counter}"
-        builder.records.stringin(
-            positions_record_name + ":NAME",
-            VAL=record_name,
-            DESC="Table of configured positional outputs",
-        ),
+        (
+            builder.records.stringin(
+                positions_record_name + ":NAME",
+                VAL=record_name,
+                DESC="Table of configured positional outputs",
+            ),
```

Comment:
Not sure why there was a trailing comma here...
```suggestion
            )
```